release/version_swap/src/main.rs

```diff
@@ -0,0 +1,61 @@
+/// Helper script to swap the version and build numbers to the ones located in .version and .version-build
```

Comment:
Instead of using a custom built tool for this and having to maintain the tool long term, we could use an already existing tool to inject the version and build numbers.

You should look into what options we have. One possibility I see is using template files and the `envsubst` command, but I'm sure there are plenty of other, possibly even smarter tools we could use instead.

In general, the more we can use existing tools/libraries and the less code we need to maintain and the more we can focus on building and maintaining features for our project.

As a general rule of thumb, we should almost always try to use preexisting tools and libraries instead of writing our own.

Reply:
++ on that. Additionally, the version bump should already be part of the RC (i.e. it should be review-able during the MR that gets tagged), to make sure it is according to our standards. 

This is because depending on the situation, we'd like to do either a patch or minor release, so making sure this change is explicit would save us a lot of trouble later down the road.